cocotb/env_info.py

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+###############################################################################
+
+class EnvInfo():
+    """Collects information about the environment in which cocotb is running"""
+    
+    def __init__(self):
+        self.sim_name    = "unknown"
+        self.sim_version = "unknown"
+        self.argv = []
+    
+    def set_sim_name(self, name):
```

Comment:
These could become ``@property`` and ``@<attr>.setter``: https://stackoverflow.com/questions/6618002/using-property-versus-getters-and-setters

Reply:
This seems like Java leaking into python to me - all these getters and setters aren't useful, let's just omit them.